### PR TITLE
[azsystemevents] Regenerating based on c25006263b7c15ad5da755a5ef864e4301e56b54

### PR DESCRIPTION
**Pull request by richardpark-msft**
Added events ACSRouterWorkerUpdatedEventData and ACSAdvancedMessageDeliveryStatusUpdatedEventData.

### CODE REVIEW

File: sdk/messaging/eventgrid/azsystemevents/internal/version.go [hunk 1]
```diff
@@ -9,5 +9,5 @@ const (
 	ModuleName = "azsystemevents"
 
 	// ModuleVersion is the semantic version (see http://semver.org) of this module.
-	ModuleVersion = "v0.2.0"
+	ModuleVersion = "v0.2.1"
```
Comment (jhendrixMSFT) [inline sdk/messaging/eventgrid/azsystemevents/internal/version.go:12]: I think this should be `v0.3.0` due to breaking changes.

File: sdk/messaging/eventgrid/azsystemevents/error.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azsystemevents
+
+// Error is an error that is included as part of a system event.
+type Error struct {
+	// Code is an error code from the service producing the system event.
+	Code string
+
+	message string
```
Comment (jhendrixMSFT) [inline sdk/messaging/eventgrid/azsystemevents/error.go:14]: Debating if this should be exported for fakes purposes?

Comment (richardpark-msft) [reply sdk/messaging/eventgrid/azsystemevents/error.go:14]: @JeffreyRichter and I had a bit of a back and forth on this. I totally don't mind it being exported. Jeff, what do you think?

Comment (JeffreyRichter) [reply sdk/messaging/eventgrid/azsystemevents/error.go:14]: I don't know enough about how fakes work with AMQP.  To me, this is like how we only expose HTTP status code and code from ResponseError but it holds on to the raw JSON error response body. This has a way of working with fakes, right? If so, I'd expect this to work the same way.

Comment (richardpark-msft) [reply sdk/messaging/eventgrid/azsystemevents/error.go:14]: Just to be clear, this isn't AMQP, it's protocol independent (the events can be received from anywhere). 

@jhendrixMSFT , thinking about this more this is the 'message' part, which is considered the cosmetic part. They can already change Code, which is really the only piece they should use.

So maybe we leave this one unexported.

Comment (jhendrixMSFT) [reply sdk/messaging/eventgrid/azsystemevents/error.go:14]: That's fine, we can always export it later if required.

File: sdk/messaging/eventgrid/azsystemevents/custom_events_test.go
Comment (jhendrixMSFT) [inline sdk/messaging/eventgrid/azsystemevents/custom_events_test.go:1]: Missing copyright header.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**